### PR TITLE
conjure-up 2.1.2 (new formula)

### PR DESCRIPTION
**Pull request by adam-stokes**
Signed-off-by: Adam Stokes <battlemidget@users.noreply.github.com>

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
This relies on https://github.com/Homebrew/homebrew-core/pull/11472, not sure if the CI bot has a dependency wait type option as that PR is Approved just waiting to be merged.

### CODE REVIEW

**Comment by battlemidget** [comment]
Now that juju-wait is merged how can I kick the ci bot off again to rerun the checks?

**Comment by arosales** [comment]
@battlemidget thanks for this formula looking forward to giving it a try in Brew.

File: Formula/conjure-up.rb [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  desc "Big software deployments so easy it's almost magical."
+  homepage "http://conjure-up.io"
+  url "https://github.com/conjure-up/conjure-up/archive/2.1.tar.gz"
+  sha256 "933f94c21b999fd24b4c304ba27c59bde9c83f478f5ee6a45cb8f265aaac2658"
+
+  head "https://github.com/conjure-up/conjure-up.git", :branch => "master"
+
+  devel do
+    url "https://github.com/conjure-up/conjure-up/archive/2.2.0-beta1.tar.gz"
+    version "2.2-beta1"
```
Comment (fxcoudert) [inline Formula/conjure-up.rb]: Is there a reason why this can't be `2.2.0-beta1`?

Comment (battlemidget) [inline Formula/conjure-up.rb]: Our product follows the same release and version scheme as https://github.com/juju/juju/releases so while not the way you or I would version, it is what I have to work with :)

**Comment by battlemidget** [comment]
Hi, is it possible to get an update on this PR? Thank you

**Comment by fxcoudert** [comment]
Thanks @battlemidget for your contribution to Homebrew!

**Pull request closed without merging**